test/unit/user_test.rb

```diff
@@ -206,10 +206,6 @@ class UserTest < ActiveSupport::TestCase
       @rubygems.first.versions.first.update! indexed: false
       assert_equal 2, @user.rubygems_downloaded.count
     end
-
-    should "total their number of pushed rubygems" do
-      assert_equal @user.total_rubygems_count, 3
-    end
```

Comment:
@sferik total_downloads_count method is only used by profile show page, so I replace it with the ```rubygems_downloaded``` method. This method also has a following test that you asked
```ruby
should "not include gem if all versions have been yanked" do
  @rubygems.first.versions.first.update! indexed: false
  assert_equal 2, @user.rubygems_downloaded.count
end
```
Do we need any other test other than this test?